ml-agents-envs/mlagents_envs/side_channel/environment_parameters_channel.py

```diff
@@ -35,3 +37,18 @@ def set_float_parameter(self, key: str, value: float) -> None:
         msg.write_int32(self.EnvironmentDataTypes.FLOAT)
         msg.write_float32(value)
         super().queue_message_to_send(msg)
+
+    def set_sampler_parameters(self, key: str, values: List[float]) -> None:
```

Comment:
Is this a public method? If it is, I do not think we should have as argument `value:List[float]`. We should have a method with a signature of type 
```python
def set_gaussian_sampler_parameter(self, key: str, mean:float, std:float) -> None:
```
and
```python
def set_uniform_sampler_parameter(self, key: str, min_value:float, max_value:float) -> None:
```

Reply:
I think this is sort of a private method because it's called on the `env_parameters` variable that is local to the `worker` function in `subprocess_env_manager.py`.  It's not currently possible to access this variable without sending requests/EnvironmentCommands. So I guess its a public method but on a limited access variable?

I'm sending a list of float because the sampler has already been encoded (into a list of floats) at this point. I didn't want info of encoding/decoding to spill outside of where it was built.

Reply:
EnvironmentParametersChannel is a public class. It is meant to be directly accessible by the user. I expect users to be able to do sampling without training if so their hearts desire.